module/Application/view/application/home/index.phtml

```diff
@@ -1,6 +1,10 @@
 <section class="intro row">
-    <div class="jumbotron bg-api-tools">
-        <h1 class="text-white">The world's easiest way to create high-quality APIs</h1>
+    <div class="jumbotron bg-api-tools" id="security-notice">
+        <h1 class="text-white">Laminas API Tools was marked as security-only</h1>
+        <p class="text-white">
+            No new features or enhancements will be developed and only security-related
+            updates will be provided going forward
+        </p>
```

Comment:
Existing content should not be overwritten, instead a new section should be added to the header, which is then displayed on all pages. And it should contain a link [to your own blog post](https://getlaminas.org/blog/2024-07-02-dotkernel-a-replacement-for-laminas-api-tools.html).

Here is an example:

![header](https://github.com/user-attachments/assets/a2fb5c69-e4b0-43c8-8c7d-e33c337ab4c7)

If we only put it on the homepage, then it doesn't help if direct links point to the documentation.

Reply:
Now it look like below 
![image](https://github.com/user-attachments/assets/fbcee325-1bf9-46f1-b3bf-152b5ab59105)